packages/vc-export/src/index.ts

```diff
@@ -9,9 +9,11 @@
  * @module @kiltprotocol/vc-export
  */
 
+export * from './fromICredential.js'
+export * as KiltCredentialV1 from './KiltCredentialV1.js'
```

Comment:
Especially because the function with the same name is exported, would it make sense to namespace them?

Reply:
I don't get it, `KiltCredentialV1` _is_ the namespace?

Reply:
Nevermind, I have probably been mistaken.